fabtests/prov/efa/Makefile.include

```diff
@@ -51,3 +51,5 @@ prov_efa_src_fi_efa_rnr_queue_resend_LDADD = libfabtests.la
 prov_efa_src_fi_efa_info_test_SOURCES = \
 	prov/efa/src/efa_info_test.c
 prov_efa_src_fi_efa_info_test_LDADD = libfabtests.la
+
+nobase_dist_config_DATA += pytest/efa/efa_device_caps.c
```

Comment:
Do you need some configure to rdma-core? since this program explicitly uses rdma-core functions.

Reply:
I switched to turn off the strict mode for rnr test in the latest revision, please review again.